docs/wiki/featurex/fortify.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: wiki
+title: fortify
+description: enables player to build fortifications.
+group: feature
+category: interaction
+parent: wiki
+mod: acex
+version:
+  major: 3
+  minor: 3
+  patch: 0
+---
+
+## 1. Overview
+
+### 1.1 Fortification
+
+Enable players to place down fortifications.
+
+### 1.2 Budget
+
+Enables mission makers / admin to restrict the amount of resources available.

```

Comment:
Does 1.1 and 1.2 really need their own titles? Just both under Overview is fine, they are so short.